_pkgdown.yml

```diff
@@ -1,3 +1,38 @@
 url: https://epiverse-trace.github.io/readepi
 template:
   package: epiversetheme
+
+reference:
```

Comment:
@Karim-Mane, could you have a look at the naming patterns and see if it can be further harmonized?

Ideally, all functions should be identifiable by a common prefix.

Reply:
@Bisaloo - Thanks for your edits and comments

I am planning to rename the functions. For example, below is how I intend to do so for Fingertips functions:

get_fingertips_metadata  --> fingertips_get_metadata
get_ind_id_from_ind_name --> fingertips_get_id_from_name
get_ind_id_from_domain_id --> fingertips_get_id_from_domain_id
get_ind_id_from_domain_name --> fingertips_get_id_from_domain_name
get_profile_name --> fingertips_get_profile_name
get_ind_id_from_profile --> fingertips_get_id_from_profile

Are you referring to something like this?

Reply:
Yes, that's the idea. You may even be able to go one step further and use [name molds](https://www.youtube.com/watch?v=z7w2lKG8zWM) instead of just prefixes. At some point, this should be documented in the package design vignette. Something like: 

"All readepi functions are named used a consistent name mold: `{HIS-name}_{action-verb}_{...}`".